### ISSUE TITLE
CA.gov logo update to meaningful <img> from deocrative.

### ISSUE THREAD

**Issue by jaredwyee** [cagov/design-system#656]
1. Coded as: Decorative
Accessible name: ca.gov logo
Change to: `<img>`

If ca.gov was removed we would only hear State of California in. It impacts the page so should be made an <img> instead of decorative. The footer can remain decorative since it would be repetitive. 
 
![image](https://user-images.githubusercontent.com/85529382/153972467-f8b109a8-6bb1-4078-bdee-880eb746c897.png)

**WCAG 2.1 Guideline**  [1.1.1 Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content)
Every image must be correctly coded as either meaningful or decorative.
If the image conveys information that isn't available through other page content, it's meaningful.
If the image could be removed from the page with no impact on meaning for function, it's decorative.

**Comment by kkoryaka** [comment]
@jaredwyee Ca.gov is actually link. It's not just the image (link has a title attribute that is read by assistive technologies) The ca.gov image inside of the link has aria-hidden=true attribute, but it's information is conveyed in the link's attribute title text. So, I am not sure if we have any problem there. 

As far as arrows on the menu, I agree, we need to make sure they have aria-hidden attributes.

**Comment by zakiya** [comment]
I think in this case the svg itself would be considered meaningful and not decorative and it would make sense to remove aria-hidden=true and add alt="Ca.gov logo" https://www.w3.org/WAI/tutorials/images/functional/. 

I agree about adding aria-hidden to the menu arrows.

**Comment by jaredwyee** [comment]
Menu being aria-hidden agreed.

For the ca.gov logo, sounds like there is the potential to 1) rely on the link being the information/description for assistive tech OR 2) have the image svg with an alt text.

It does not seem like we need BOTH. Is there a better approach for which one? @zakiya @kkoryaka Or please clarify on why we might need both 😁

**Comment by zakiya** [comment]
I would prefer to use an alt tag on the logo and remove the the title attribute from the a tag. https://www.a11yproject.com/posts/title-attributes/ https://www.deque.com/blog/text-links-practices-screen-readers/ We could also ask the CDT Accessibility Engineer since she was the original reporter.

**Comment by jaredwyee** [comment]
I'm good with proceeding as you stated: use alt tag on logo and remove the title attribute.

Original recommendation from Tiffany was the <img>, with  alt tag anyways.

**Comment by jaredwyee** [comment]
Confirmed working. Closing. Pending move to Done once release merge complete.
